modules/apps/foundation/frontend-theme/frontend-theme-classic/src/css/_portlet_hello_world.scss

```diff
@@ -8,6 +8,7 @@
 	.portlet-title-text {
 		color: $bright-color;
 		font-size: 300%;
+		line-height: initial;
```

Comment:
This should be applied for all portlets instead of just the hello world portlet.  Please see https://github.com/brianchandotcom/liferay-portal/pull/46097/commits/5a4851aeec65495ea407be88a78fb755c1f1fbdb for corrected fix.